clients/docs/csharp.rst

```diff
@@ -0,0 +1,142 @@
+.. _client-examples-csharp:
+
+.NET
+====
+
+In this tutorial, you will produce messages to and consume messages from an |ak-tm| cluster using the .NET Producer and Consumer.
+
+
+Prerequisites
+-------------
+
+Client
+~~~~~~
+
+-  `.NET Core 2.1 <https://dotnet.microsoft.com/download>`__ or higher to run the demo application
+
+-  On Windows, default trusted root CA certificates are stored in the Windows Registry. These are required for secure access to Confluent Cloud. The .NET library does not currently have the capability to access these certificates, so you must obtain them from somewhere else, for example use the ``cacert.pem`` file distributed with curl (`download cacert.pm <https://curl.haxx.se/ca/cacert.pem>`__).
+
+Kafka Cluster
+~~~~~~~~~~~~~
+
+.. include:: includes/client-example-prerequisites.rst
+
+
+Setup
+-----
+
+#. Clone the `confluentinc/examples GitHub repository <https://github.com/confluentinc/examples>`__ and check out the :litwithvars:`|release|-post` branch.
+
+   .. codewithvars:: bash
+
+      git clone https://github.com/confluentinc/examples
+      cd examples
+      git checkout |release|-post
+
+#. Change directory to the example for .NET.
+
+   .. sourcecode:: bash
+
+      cd clients/cloud/csharp/
+
+#. .. include:: includes/client-example-create-file.rst
+
+
+Example 1: Basic Produce and Consume
+------------------------------------
+
+In this example, the producer writes records to a topic in your |ak|
+cluster. Each record has a key representing a username (e.g. ``alice``)
+and a value of a count, formatted as json (e.g. ``{"count": 0}``). The
+consumer reads the same topic and keeps a rolling sum of the counts as
+it processes each record.
+
+Produce Records
+~~~~~~~~~~~~~~~
+
+#. Build the client example application
+
+   .. code:: shell
+
+       dotnet build
+
+#. Run the example application, passing in arguments for:
+ 	
+   - (a) whether to produce or consume (produce)
```

Comment:
@joel-hamill is this the desired rendering?  (note bullet _and_ letters)

<img width="659" alt="Screen Shot 2020-08-01 at 12 06 26 PM" src="https://user-images.githubusercontent.com/12448600/89105483-86c62b00-d3ef-11ea-9b41-1b671eefaa28.png">

Reply:
Outdated.